store/snap_remote_repo.go

```diff
@@ -108,8 +110,12 @@ func getStructFields(s interface{}) []string {
 	return fields
 }
 
+func useStagingCpi() bool {
```

Comment:
I think this should useStagingCPI per conventions, also actually may just be called useStagingStore? given how it is used?